k8s/data-warehouse/stage-data-warehouse-api.yaml

```diff
@@ -82,70 +82,6 @@ spec:
   minReplicas: 1
   maxReplicas: 2
   metrics:
-  - type: Resource
-    resource:
-      name: cpu
-      target:
-        type: Utilization
-        averageUtilization: 80
-
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: airqo-stage-datawarehouse-msg-broker
-  namespace: staging
-spec:
-  selector:
-    matchLabels:
-      app: sta-datawarehouse-msg-broker
-  replicas: 2
-  strategy:
-    type: RollingUpdate
-    rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 0
-  minReadySeconds: 5
-  template:
-    metadata:
-      labels:
-        app: sta-datawarehouse-msg-broker
-    spec:
-      containers:
-        - name: msg-broker
-          image: us.gcr.io/airqo-250220/airqo-stage-datawarehouse-msg-broker:latest
-          imagePullPolicy: Always
-          resources:
-            requests:
-              cpu: 100m
-              memory: 250Mi
-            limits:
-              cpu: 200m
-              memory: 700Mi
-          envFrom:
-            - configMapRef:
-                name: stage-datawarehouse-api-config
-          volumeMounts:
-          - name: config-volume
-            mountPath: /etc/config
-      volumes:
-        - name: config-volume
-          configMap:
-            name: stage-datawarehouse-api-config-files
-
----
-kind: HorizontalPodAutoscaler
-apiVersion: autoscaling/v2
-metadata:
-  name: airqo-stage-datawarehouse-msg-broker-hpa
-spec:
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: Deployment
-    name: airqo-stage-datawarehouse-msg-broker
-  minReplicas: 1
-  maxReplicas: 2
-  metrics:
```

Comment:
Hi @123MwanjeMike looks like you are removing the deployment manifest for the message broker but you did not delete the entire metircs section..

Reply:
Hi @NoahNsimbe 
Those are metrics for data warehouse deployment hpa.
https://github.com/airqo-platform/AirQo-api/blob/b4b99a61557b5bae7ca46708711cf3cdbd3ac040/k8s/data-warehouse/stage-data-warehouse-api.yaml

Reply:
Great. Thanks